pick.1

```diff
@@ -32,6 +32,8 @@ Both parts will be displayed but only the first part will be used when
 searching.
 .It Fl h
 Output a help message and exit.
+.It Fl m
```

Comment:
Why is this feature opt-in to begin with?

Reply:
Without -m (multiple choices), the caller waits for one line string. in 'multiple choices'  mode the caller is prepared to receive a list of lines. So he had to specify which mode wants.